docs/components/toggle/toggle-demo.js

```diff
@@ -0,0 +1,20 @@
+import Util from '../../_util';
+
+if (document.getElementById('vue-toggleDemo')) {
+    new Vue({
+        el: '#vue-toggleDemo',
+        computed: {
+            snippet: function () {
+                return Util.snippet(`
+                    <hx-toggle-control class="switch">
```

Comment:
In the same context as the Switch component, we should not need any classes to render the default Toggle component.